### PR TITLE
Broadcast

### PR DESCRIPTION
**Pull request by MarcoPolo**
First mega hack version of broadcast :P

### CODE REVIEW

File: app/js/player.js [hunk 1]
```diff
@@ -1,27 +1,118 @@
...
+  componentWillMount: function(){
+    this.props.isFollowingStream.observe(function(isFollowing){
+      this.setState({isFollowing:isFollowing});
+    }.bind(this));
+
+    this.props.isBroadCastingStream.observe(function(isBroadcasting){
+      this.setState({isBroadcasting: isBroadcasting});
+    }.bind(this));
+
+    var broadcastingAndBroadcastNameStream = mergeStreams(this.props.broadcastNameStream, this.props.isBroadCastingStream,
```
Comment (L1fescape) [inline app/js/player.js:38]: Why are these streams being merged?

File: app/js/init.js [hunk 2]
```diff
@@ -2,8 +2,10 @@
 
 var React = require('react');
 var Player = require('./player');
+var simpleStream = require('simple-stream');
+var Stream = simpleStream.Stream;
 
 React.render(
-  <Player />,
+  <Player isFollowingStream={Stream()} broadcastNameStream={Stream()} isBroadCastingStream={Stream()} />,
```
Comment (L1fescape) [inline app/js/init.js:9]: I'd prefer these properties not be set on the Player directly. There should be an API for interacting with the player which controls the state of playback. I'll try and spec that out tonight.

File: app/js/player.js (hunk 1 above)
Comment (MarcoPolo) [inline app/js/player.js:38]: It's so we only emit a name when we are broadcasting, otherwise null

File: app/js/init.js (hunk 2 above)
Comment (MarcoPolo) [inline app/js/init.js:9]: Of course :)

**Comment by MarcoPolo** [comment]
Hacks on hacks and not very good or reusable. Closing for now

**Pull request closed without merging**